client/webserver/site/src/js/order.js

```diff
@@ -30,9 +30,8 @@ export default class OrderPage extends BasePage {
     ])
 
     if (page.cancelBttn) {
-      Doc.bind(page.cancelBttn, 'click', () => {
-        this.showForm(page.cancelForm)
-      })
+      Doc.bind(page.cancelBttn, 'click', () =>
+        this.showForm(page.cancelForm))
```

Comment:
Please undo the re-formatting in this file. If you really feel it's important to reformat this file, submit a separate PR please. But in all honesty, nothing here needs reformatting.